.releaserc.json

```diff
@@ -0,0 +1,71 @@
+{
+  "branches": ["main"],
+  "plugins": [
+    [
+      "@semantic-release/commit-analyzer",
+      {
+        "preset": "angular",
+        "releaseRules": [
+          {"type": "docs", "scope": "README", "release": "patch"},
+          {"type": "refactor", "release": "patch"},
+          {"type": "style", "release": "patch"},
+          {"type": "chore", "scope": "deps", "release": "patch"},
+          {"type": "perf", "release": "patch"},
+          {"type": "test", "release": false},
+          {"type": "ci", "release": false},
+          {"type": "build", "release": false}
+        ]
+      }
+    ],
+    [
+      "@semantic-release/release-notes-generator",
+      {
+        "preset": "angular",
+        "presetConfig": {
+          "types": [
+            {"type": "feat", "section": "Features"},
+            {"type": "fix", "section": "Bug Fixes"},
+            {"type": "perf", "section": "Performance Improvements"},
+            {"type": "refactor", "section": "Code Refactoring"},
+            {"type": "docs", "section": "Documentation"},
+            {"type": "style", "section": "Styles"},
+            {"type": "chore", "section": "Maintenance", "hidden": true},
+            {"type": "test", "section": "Tests", "hidden": true},
+            {"type": "ci", "section": "CI", "hidden": true},
+            {"type": "build", "section": "Build", "hidden": true}
+          ]
+        },
+        "writerOpts": {
+          "commitsSort": ["subject", "scope"]
+        }
+      }
+    ],
+    [
+      "@semantic-release/changelog",
+      {
+        "changelogFile": "CHANGELOG.md"
+      }
+    ],
+    [
+      "@semantic-release/npm",
+      {
+        "npmPublish": true
+      }
+    ],
+    [
+      "@semantic-release/git",
+      {
+        "assets": ["CHANGELOG.md", "package.json"],
+        "message": "chore(release): ${nextRelease.version} [skip ci]\n\n${nextRelease.notes}"
+      }
+    ],
+    [
+      "@semantic-release/github",
+      {
+        "successComment": false,
```

Comment:
I would probably suggest having this set to true, those success comments can be really nice when you're trying to track down what version a particular change went into.